app/src/test/java/waiter/RouterTest.java

```diff
@@ -29,4 +29,16 @@ void simpleGetWithBodyRoute() {
         assertEquals("Content-Length: 11", route.getHeaders());
         assertEquals("Hello world", route.getBody());
     }
+
+    @Test
+    void headSimpleGetReturnsRouteWithoutBody() {
```

Comment:
Should we add a test for no URL found?

Reply:
Sure! Ill add a test for both 500 and 404